migtests/tests/oracle/assessment-report-test/oracle_assessment_report.sql

```diff
@@ -0,0 +1,233 @@
+-- VOYAGER UNSUPPORTED ORACLE INDEXES examples
```

Comment:
@sanyamsinghal pls go through this to see if we have missed any case

Reply:
I remember bitmap indexes being supported, as they as converted to GIN or some other index?
cc @shubham-yb @makalaaneesh

Reply:
Yes BITMAP indexes are converted to GIN. We post a note in case we encounter them since they are partially supported. Added this to verify if the note is properly populated or not in this case.
Will add a comment to the script.

Reply:
yeah just add a small comment here, about how BITMAP indexes are supported/unsupported